### ISSUE TITLE
Undefined index: xxx in phalcon/mvc/model.zep on line 2092

### ISSUE THREAD

**Issue by linauror** [phalcon/cphalcon#12709]
> **When I insert the database, filed is empty,the following error is displayed.**
```php
Notice: Undefined index: shop_name in phalcon/mvc/model.zep on line 2092
```
Other fields will have this situation

And MetaData is closed

The mysql table field is 
```mysql
`shop_name` VARCHAR(50) NOT NULL COMMENT 'xxx',
```

The php code is
```php
$this->create(['shop_name' => '']);
```
It is possible to modify the database fields as follows

```mysql
`shop_name` VARCHAR(50) NOT NULL DEFAULT '' COMMENT 'xxx',
```
### Details

* Phalcon version: 3.0.4
* PHP Version: 5.5.38
* Operating System: windows
* Installation type: dll
* Server: Apache
* Other related info (Database, table schema): mysql InnoDB

**Comment by Jurigag** [comment]
It's fixed on 3.1.x branch.

**Comment by linauror** [comment]
Thanks
